src/main/java/com/t4t/thought4thought/entities/User.java

```diff
@@ -1,20 +1,31 @@
 package com.t4t.thought4thought.entities;
 
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
+import javax.persistence.*;
 
-@Entity // This tells Hibernate to make a table out of this class
+@Entity
 public class User {
     @Id
     @GeneratedValue(strategy=GenerationType.AUTO)
     private Integer id;
 
-    private String name;
+    private String firstName;
+
+    private String lastName;
 
     private String email;
 
+    private String password;
+
+    private String passwordSalt;
+
+    private String userType;
+
+    private String interests;
```

Comment:
The user may have more than one interest. Are they being stored as a string and using a delimiter to separate them?

Reply:
Good catch - this is something we'll have to get back to when the mockups are enhanced. Right now it's just a simple text input field which doesn't really conform the user to any specific way of entering more than one input, as opposed to say a dropdown where we can pass the values selected through a comma separated string.